peer/listener.go

```diff
@@ -125,7 +125,7 @@ func (lstn *listener) Run(args interface{}, shutdown <-chan struct{}) {
 				case lstn.socket6:
 					lstn.process(lstn.socket6)
 				case lstn.sigReceive:
-					s.RecvMessageBytes(0)
+					_, _ = s.RecvMessageBytes(0)
```

Comment:
For what?

Reply:
If you use linter, it will complain about this line